packages/toggle/__tests__/Toggle.cy.tsx

```diff
@@ -12,32 +12,29 @@ describe('Toggle', () => {
   it('is accessible', () => {
     cy.mount(
       <>
-        <label htmlFor="toggle">label</label>
-        <Toggle id="toggle" />
+        <Toggle aria-label="toggle" />
       </>
     );
     cy.checkA11y();
   });
 
   it('can be reached with the keyboard', () => {
-    cy.mount(<Toggle />);
+    cy.mount(<Toggle defaultSelected={false} />);
 
     cy.getByTestId(TOGGLE).focus();
     cy.getByTestId(TOGGLE).type(' ');
-    cy.getByTestId(TOGGLE).should('be.checked');
```

Comment:
there should probably still be an assertion in here, is `not.be.checked` what you're after?